client/src/views/statistics/statistics.module.scss

```
@@ -0,0 +1,62 @@
+@import '../../core/styles/color.scss';
```

```suggestion
@import '@/core/styles/color.scss';
```

.scss에서도 webpack의 resolve.alias를 이용할 수 있습니다.

---

맞네요.. 감사합니다~!